### PR TITLE
Tools: Added support for single file in api-tree task.

### PR DESCRIPTION
**Pull request by bre1470**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207121729234896

### CODE REVIEW

**Review by goransle — APPROVED** [review]
Looks good

File: tools/gulptasks/api-tree.js [hunk 1]
```diff
@@ -64,18 +64,36 @@ async function apiTree() {
...
-        ));
-    const optionFiles = FSLib
-        .getFilePaths((sources || 'ts'), true)
-        .filter(path => (
-            path.endsWith('Options.d.ts') ||
-            path.endsWith('Options.ts')
-        ));
+    let singleFile = false;
+    let moduleSource = Path.join('ts', 'Core');
+    let optionSource = 'ts';
```
Comment (hubertkozik) [inline tools/gulptasks/api-tree.js:69]: Should we follow the syntax with variables in new lines with commas?

**Review by hubertkozik — APPROVED** [review]
One question for curiosity.

**Review by marvin19 — APPROVED** [review]

File: tools/gulptasks/api-tree.js (hunk 1 above)
Comment (bre1470) [reply tools/gulptasks/api-tree.js:69]: Good question. The comma approach was back then enforced on Highcharts code, but not on tooling. I personally prefer separated statements over the list, but you could escalate it and I would fix it. 😅

Comment (hubertkozik) [reply tools/gulptasks/api-tree.js:69]: I'm fine with separated statements 😁

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**